@navikt/core/tokens/config/build.ts

```diff
@@ -1,9 +1,26 @@
-module.exports = {
+import StyleDictionary from "style-dictionary";
+import {
+  PlatformConfig,
+  Transform,
+  TransformedToken,
+} from "style-dictionary/types";
+import { kebabCase } from "./kebabCase";
+
+const V1_DIST = "dist/";
+
+const kebabTransform: Transform = {
+  name: "name/cti/kebab",
+  type: "name",
+  transform: (token: TransformedToken, options: PlatformConfig) =>
+    kebabCase([options.prefix].concat(token.path).join(" ")),
+} as const;
```

Comment:
Does `as const` have any effect?
```suggestion
};
```

Reply:
Yes, i'm getting type-errors when not added since style-dictionary don't understand that `type` is "name" instead of "string" then